examples/minimal/src/pages/index.astro

```diff
@@ -5,7 +5,6 @@
 <html lang="en">
 	<head>
 		<meta charset="utf-8" />
-		<link rel="icon" type="image/x-icon" href="/favicon.ico" />
```

Comment:
is there no image at public/favicon.ico?

Reply:
There is an image, and it will get picked up automatically. All the way back to Internet Explorer 5. https://en.wikipedia.org/wiki/Favicon#History